### ISSUE TITLE
Typo to be corrected related to Assumption Trust

### ISSUE THREAD

**Issue by GSMA-EIG** [eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework#238]
§ 6.1.3: “Relying Parties may try to request attributes from a Wallet Instance for which they have lawful grounds”. We think that there is a typo. The sentence shall be “Relying Parties may try to request attributes from a Wallet Instance for which they have no lawful grounds”.

**Comment by skounis** [comment]
@GSMA-EIG Indeed, the typo has been corrected in the main branch and will be included in the upcoming service release.

* https://github.com/eu-digital-identity-wallet/eudi-doc-architecture-and-reference-framework/commit/90254c7045b3ee9a7a97a42a696f5d9389e323dc

**Comment by phin10** [comment]
Closed after release of ARF 1.10.0
